bundles/core/src/main/java/com/adobe/cq/forms/core/components/models/aemform/AEMForm.java

```diff
@@ -40,7 +40,7 @@ public interface AEMForm extends Component {
      * Defines the form type
      */
     enum FormType {
-        NO_FORM_SELECTED, MOBILE_FORM, ADAPTIVE_FORM, MC_DOCUMENT, MOBILE_FORMSET
+        NO_FORM_SELECTED, MOBILE_FORM, ADAPTIVE_FORM, MC_DOCUMENT, MOBILE_FORMSET, ADAPTIVE_FORM_V2
```

Comment:
FormType should be ADAPTIVE_FORM only. 
version would be the differentiator.

Reply:
exposed getVersion , so version check is added.